packages/firestore/src/core/query.ts

```diff
@@ -434,12 +463,12 @@ export class RelationOp {
   }
 }
 
-export class RelationFilter implements Filter {
+export class RelationFilter extends Filter {
   constructor(
```

Comment:
You can omit constructors that match the super class constructor (here and everywhere)..

Reply:
Don't you need to keep them to ensure (in this case) field, op, and value are present? Or in other words, these ctors don't match the parent ctor.

Reply:
Oh, yes, that's true. The constructors are much more powerful at second glance.